pyslim/slim_tree_sequence.py

```diff
@@ -627,7 +627,7 @@ def individuals_alive_at(self, time, stage='late', remembered_stage=None,
         :param str stage: The stage in the SLiM life cycle that we are inquiring
             about (either "early" or "late"; defaults to "late").
         :param str remembered_stage: The stage in the SLiM life cycle
-            that individuals were Remembered during (defaults to the stage the
+            that individuals were Remembered or Retained (defaults to the stage the
```

Comment:
maybe omit "or Retained" here? Since one retains by calling treeSeqRemember, it seems sufficient; and otherwise it suggests there is a treeSeqRetain method?